interpretation/ww_head.py

```diff
@@ -15,6 +15,17 @@ def process(message, isGameMaster = False):
     message_channel = message.channel.id
     user_role = db_get(user_id,'role')
 
+    '''testcc'''
+    # This function is merely a temporary one, to test if the cc creation command is working properly.
+    if is_command(message,['cc','testcc','test_cc']):
+        members = check.users(message)
+        name = message.content.split(' ')[1]
+        return Mailbox().create_cc(name,user_id,members)
+    if is_command(message,['cc','testcc','test_cc'],True):
+        msg = "**Usage:** `" + prefix + "cc <name> <user> <user> <user> ...`\n\nExample: `" + prefix + "cc the_cool_ones @Randium#6521`"
```

Comment:
This is gonna ping you

Reply:
No, it isn't. It's inbetween `-brackets.

Look! `Oi! @Randium#6521 dude, yeah you! @BenTechy66#8809 and his buddy think this'll ping them!`

Reply:
Oh, well, anyway. It won't.